wasm/jit/jit_amd64_test.go

```diff
@@ -3985,6 +4000,136 @@ func TestAmd64Compiler_compileF64PromoteFromF32(t *testing.T) {
 	}
 }
 
+func TestAmd64Compiler_compileITruncFromF(t *testing.T) {
+	for _, tc := range []struct {
+		outputType wazeroir.SignedInt
+		inputType  wazeroir.Float
+	}{
+		{outputType: wazeroir.SignedInt32, inputType: wazeroir.Float32},
+		{outputType: wazeroir.SignedInt32, inputType: wazeroir.Float64},
+		{outputType: wazeroir.SignedInt64, inputType: wazeroir.Float32},
+		{outputType: wazeroir.SignedInt64, inputType: wazeroir.Float64},
+		{outputType: wazeroir.SignedUint32, inputType: wazeroir.Float32},
+		{outputType: wazeroir.SignedUint32, inputType: wazeroir.Float64},
+		{outputType: wazeroir.SignedUint64, inputType: wazeroir.Float32},
+		{outputType: wazeroir.SignedUint64, inputType: wazeroir.Float64},
+	} {
+		tc := tc
+		t.Run(fmt.Sprintf("%s from %s", tc.outputType, tc.inputType), func(t *testing.T) {
+			for _, v := range []float64{
+				0, 100, -100, 1, -1,
+				100.01234124, -100.01234124, 200.12315,
+				6.8719476736e+10, /* = 1 << 36 */
+				-6.8719476736e+10,
+				1.37438953472e+11, /* = 1 << 37 */
+				-1.37438953472e+11,
+				math.MinInt32,
+				// Note that math.MaxInt32 is rounded up to math.MaxInt32+1 in 32-bit float representation.
+				// See the output of "fmt.Printf("%f\n", float32(2147483647))".
```

Comment:
Instead of "See the output" here and elsewhere, which makes the reader do more work. Maybe switch to.. for example, fmt.Sprintf(...) == "this" not "that"

Reply:
https://github.com/tetratelabs/wazero/pull/124/commits/02a1441a90acfe0a2eee6d211f17b733199c02d0 done